verifier/verifier.go

```diff
@@ -63,6 +66,25 @@ func New(trustPolicy *trustpolicy.Document, trustStore truststore.X509TrustStore
 	}, nil
 }
 
+// NewWithRevocationClient creates a new verifier given trustPolicy, trustStore, pluginManager, and revocationClient
+func NewWithRevocationClient(trustPolicy *trustpolicy.Document, trustStore truststore.X509TrustStore, pluginManager plugin.Manager, revocationClient *http.Client) (notation.Verifier, error) {
+	if revocationClient == nil {
+		revocationClient = &http.Client{Timeout: 5 * time.Second}
+	}
```

Comment:
Why to default when you are taking revocationClient as input? Infact we should add this defualt in `New` method (revocation implementation is not optional). 
```suggestion
	if revocationClient == nil {
		return nil, errors.new('revocaton client is nil')
	}
```